src/main/java/com/squareup/kotlinpoet/ParameterSpec.kt

```diff
@@ -152,6 +153,8 @@ class ParameterSpec private constructor(builder: ParameterSpec.Builder) {
     @JvmStatic fun unnamed(type: Type) = unnamed(type.asTypeName())
 
     @JvmStatic fun unnamed(type: TypeName) = Builder("", type).build()
+
+    @JvmStatic fun untyped(name: String) = Builder(name, null).build()
```

Comment:
I'm a bit skeptical about needing to support this. Can we leave it out and see if anyone ever complains?

Reply:
Sure